internal/ingress/controller/kong.go

```diff
@@ -148,7 +148,7 @@ func (n *NGINXController) syncGlobalPlugins() error {
 	// plugins in Kong
 	currentGlobalPlugins := make(map[string]kong.Plugin)
 	for _, plugin := range plugins {
-		if isEmpty(plugin.RouteID) && isEmpty(plugin.ServiceID) && isEmpty(plugin.ConsumerID) {
```

Comment:
is this and the following several changes needed because the api change in the upstream go-kong library?